.pre-commit-config.yaml

```diff
@@ -25,7 +25,12 @@ repos:
       rev: v1.4.0
       hooks:
       -   id: detect-secrets
-          args: [ '--baseline', '.secrets.baseline' ]
+          args: [
```

Comment:
Did we want to stick with using the inline exclude marker in the yaml instead?
`# pragma: allowlist secret`

Reply:
I guess I don't see a reason to do that. The word `inherit` is not a secret, and we can automatically ignore the many uses of it that we'll end up with without having to remember to add this inline marker. The change here and the python file are not onerous things to maintain in my opinion.

Reply:
But if you feel strongly, go ahead and make the change.

Reply:
Updated and verified